selftests/functional/test_loader.py

```diff
@@ -233,11 +230,11 @@ def test_runner_simple_python_like_multiple_files(self):
         cmd_line = "%s list -V %s" % (AVOCADO, mytest)
         result = process.run(cmd_line)
         self.assertIn('SIMPLE: 1', result.stdout)
-        # job should be able to finish under 5 seconds. If this fails, it's
-        # possible that we hit the "simple test fork bomb" bug
+        # Job should finish immediately unless there is a bug in Avocado
+        # which produces a fork-bomb.
```

Comment:
Still don't understand. Immediately could mean anything, on loaded system executing `/bin/true` can take an hour and it would not be because of a fork bomb. The point I keep raising is that the 5s timeout is not sufficient for heavy loaded systems, we were not expecting that as it is 10 times execution time on our systems so the fix is not to disable the test, but to increase the timeout.